sdk/typescript/src/types/transactions.ts

```diff
@@ -180,6 +180,12 @@ export type TransactionQuery =
 
 export type Ordering = 'Ascending' | 'Descending';
 
+export type Owner =
```

Comment:
This has been defined in https://github.com/MystenLabs/sui/blob/d8633441e79f3d30aba4b001de94808c294b4317/sdk/typescript/src/types/common.ts#L10 ? But there seems to be an outdated line https://github.com/MystenLabs/sui/blob/d8633441e79f3d30aba4b001de94808c294b4317/sdk/typescript/src/types/common.ts#L13 could you help remove it?